apps/meteor/server/services/omnichannel-voip/service.ts

```diff
@@ -267,87 +268,75 @@ export class OmnichannelVoipService extends ServiceClassInternal implements IOmn
 		return this.voipRoom.findOneByIdAndVisitorToken(rid, token, { projection });
 	}
 
-	private async calculateOnHoldTimeForRoom(room: IVoipRoom, closedAt: Date): Promise<number> {
-		if (!room || !room.callUniqueId) {
-			return 0;
-		}
-
-		const events = await this.pbxEvents.findByEvents(room.callUniqueId, ['Hold', 'Unhold']).toArray();
-		if (!events.length) {
-			// if there's no events, that means no hold time
-			return 0;
-		}
-
-		if (events.length === 1 && events[0].event === 'Unhold') {
-			// if the only event is an unhold event, something bad happened
-			return 0;
-		}
-
-		if (events.length === 1 && events[0].event === 'Hold') {
-			// if the only event is a hold event, the call was ended while on hold
-			// hold time = room.closedAt - event.ts
-			return closedAt.getTime() - events[0].ts.getTime();
-		}
-
-		let currentOnHoldTime = 0;
-
-		for (let i = 0; i < events.length; i += 2) {
-			const onHold = events[i].ts;
-			const unHold = events[i + 1]?.ts || closedAt;
-
-			currentOnHoldTime += unHold.getTime() - onHold.getTime();
-		}
-
-		return currentOnHoldTime;
-	}
-
-	// Comment can be used to store wrapup call data
 	async closeRoom(
 		closerParam: ILivechatVisitor | ILivechatAgent,
 		room: IVoipRoom,
 		user: IUser,
-		comment?: string,
-		tags?: string[],
+		options: { comment?: string; tags?: string[] },
```

Comment:
Should options be optional? 👀

Reply:
I see no issues with that, let me do that...